redhat-cop/casl-ansible
# Openshift mgmt

What does this PR do?

added support for pruning projects based on age in hours
How should this be manually tested?

See README in the openshift-management role
Is there a relevant Issue open for this?

N/A
Who would you like to review this?

cc: @redhat-cop/casl @sabre1041
